lib/collections.js

```diff
@@ -15,9 +15,12 @@ var paragraph =
 // The word list normalized (all lower case with punctuation removed)
 var normalizedWords = [];
 
-/* your code starts here */
-
-/* your code ends here */
+// Split the words
+var words = paragraph.split(/\s+/);
+// Remove the puncuation and caps
+for (var i = 0; i < words.length; i++) {
+  var normalWord = words[i].replace(/[^\w’]/, '').toLowerCase();
+  normalizedWords[i] = normalWord;
```

Comment:
This solution looks right. Unfortunately, it seems that you forgot to close your `for` loop - notice how the curly brace in line 21 is doesn't have a partner? That would prevent your code from running, and cause Node to spit back an error.